script/1_0_2current_en_script/Common/00_00.txt

```diff
@@ -46,7 +46,7 @@ I got that she was in a rush, but she really shouldn't be running in a hospital.
 <Page13>
 "――――――"
 I couldn't raise my body, so I turned my head to look up at the window.
-My eyeballs throbbed as though they were about to split into two.
+My eyeballs throbbed as though they were about to split in two.
```

Comment:
"into two" is a bit more awkward to say, and seems to be less prevalent

![2021-12-02-002422_1994x874_scrot](https://user-images.githubusercontent.com/2476799/144362792-d8f38355-9880-4836-b4dd-df0e8a90e933.png)